models/articles.model.js

```diff
@@ -40,3 +40,22 @@ ORDER BY
     throw err;
   }
 };
+
+exports.updateArticle = async (article_id, inc_votes) => {
+  try {
+    let updateQuery = `UPDATE
+    articles
+SET
+    votes = votes + $1
+WHERE
+    article_id = $2
+    RETURNING *;;`;
+    const { rows } = await db.query(updateQuery, [inc_votes, article_id]);
+    if (rows[0].length === 0) {
+      throw { status: 404, msg: "article does not exist" };
+    }
+    return rows[0];
+  } catch (error) {
+    throw error;
```

Comment:
You don't need a catch block on your controller and your model. Remember that your model essentially returns a Promise chain and it that doesn't resolve it will be caught by the catch block on the controller anyway and then sent to the error handling middleware 👍